test_runner/regress/test_compatibility.py

```diff
@@ -261,7 +261,10 @@ def prepare_snapshot(
     # Update paths and ports in config files
     pageserver_toml = repo_dir / "pageserver.toml"
     pageserver_config = toml.load(pageserver_toml)
-    pageserver_config["remote_storage"]["local_path"] = str(repo_dir / "local_fs_remote_storage")
+    # FIXME: this needs to be changed if the path changes
+    pageserver_config["remote_storage"]["local_path"] = str(
+        repo_dir / "local_fs_remote_storage" / "pageserver"
+    )
```

Comment:
Can we use something like `#remote_storage.root` here?
If it requires some non-trivial changes, I'm ok to leave this `#FIXME` for now

Reply:
The "pageserver" component (the `RemoteStorageKind.configure(..., user: str, ...)` arg) comes from NeonEnvBuilder so I don't see an elegant way to do this.

Perhaps if we would have `LocalFsStorage.path_for_component(repo_dir: Path, user: str)` then this would work.

Reply:
> Perhaps if we would have LocalFsStorage.path_for_component(repo_dir: Path, user: str) then this would work.

This sounds nice. We could add it later.

Reply:
[eaa4c82](https://github.com/neondatabase/neon/pull/5243/commits/eaa4c82064a4b8e8737668c97e1d6dadebf85b24)